stubs/app/Actions/Jetstream/CreateTeam.php

```diff
@@ -27,9 +27,14 @@ public function create($user, array $input)
 
         AddingTeam::dispatch($user);
 
-        return $user->ownedTeams()->create([
+        $team = $user->ownedTeams()->create([
             'name' => $input['name'],
             'personal_team' => false,
         ]);
+
+        $user->current_team_id = $team->id;
```

Comment:
Great thanks!

Reply:
@OscarKolsrud my apologies, was typing on my phone, the curly braces should be square brackets

Reply:
Yep, just noticed! Fixed in a seperate commit :) In my opinion line 150 from TeamBehaviourTest can be removed since there really is no need for it. Just tested this fix on a inertia stack and everything seems to update like it should in the UI